WordPress/src/test/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/TestContent.kt

```diff
@@ -659,6 +660,22 @@ $newRefactoredGalleryBlockInnerBlocks</figure>
 <figure class="wp-block-audio"><audio controls src="$remoteAudioUrl"></audio></figure>
 <!-- /wp:audio -->"""
 
+    const val oldVideoPressBlockWithDefaultAttrs = """<!-- wp:videopress/video {"id":${localMediaId}} -->
+<figure class="wp-block-videopress-video wp-block-jetpack-videopress jetpack-videopress-player"></figure>
+<!-- /wp:videopress/video -->"""
+
+    const val newVideoPressBlockWithDefaultAttrs = """<!-- wp:videopress/video {"id":${remoteMediaId},"guid":"${videoPressGuid}"} -->
+<figure class="wp-block-videopress-video wp-block-jetpack-videopress jetpack-videopress-player"><div class="jetpack-videopress-player__wrapper">https://videopress.com/v/${videoPressGuid}?resizeToParent=true&amp;cover=true&amp;preloadContent=metadata&amp;useAverageColor=true</div></figure>
+<!-- /wp:videopress/video -->"""
+
+    const val oldVideoPressBlockWithAttrs = """<!-- wp:videopress/video {"autoplay":true,"controls":false,"description":"","id":${localMediaId},"loop":true,"muted":true,"playsinline":true,"poster":"https://test.files.wordpress.com/2022/02/265-5000x5000-1.jpeg","preload":"none","seekbarColor":"#abb8c3","seekbarLoadingColor":"#cf2e2e","seekbarPlayedColor":"#9b51e0","title":"Demo title","useAverageColor":false} -->
+<figure class="wp-block-videopress-video wp-block-jetpack-videopress jetpack-videopress-player"></figure>
+<!-- /wp:videopress/video -->"""
```

Comment:
I realized that the typical case when a VideoPress block already has attributes is a video replacement. In that case, the HTML also contains the `div` element, e.g.: 
```
<!-- wp:videopress/video {"autoplay":true,"controls":false,"description":"","id":${localMediaId},"loop":true,"muted":true,"playsinline":true,"poster":"https://test.files.wordpress.com/2022/02/265-5000x5000-1.jpeg","preload":"none","seekbarColor":"#abb8c3","seekbarLoadingColor":"#cf2e2e","seekbarPlayedColor":"#9b51e0","title":"Demo title","useAverageColor":false} -->
<figure class="wp-block-videopress-video wp-block-jetpack-videopress jetpack-videopress-player"><div class="jetpack-videopress-player__wrapper">
https://videopress.com/v/AbCdE?resizeToParent=true&amp;cover=true&amp;autoPlay=true&amp;controls=false&amp;loop=true&amp;muted=true&amp;persistVolume=false&amp;playsinline=true&amp;posterUrl=https%3A%2F%2Ftest.files.wordpress.com%2F2022%2F02%2F265-5000x5000-1.jpeg&amp;preloadContent=none&amp;sbc=%23abb8c3&amp;sbpc=%239b51e0&amp;sblc=%23cf2e2e
</div></figure>
<!-- /wp:videopress/video -->
```

Maybe we should use this HTML for this test case, WDYT?

Reply:
Solving this thread as I couldn't reproduce the issue again ([reference](https://github.com/wordpress-mobile/WordPress-Android/pull/18187#pullrequestreview-1366545133)).